### ISSUE TITLE
Crunchyroll login with credentials is broken.

### ISSUE THREAD

**Issue by EricJSimon** [streamlink/streamlink#3159]
## Plugin Issue

- [x] This is a plugin issue and I have read the contribution guidelines.

### Description

Whenever I try to use my username and password to login with streamlink I get an error stating: **error: Unable to validate API response: Unable to validate key 'auth': Type of None should be 'str' but is 'NoneType'**. 
This does not happen when I start a stream without using my credentials.
I have used '--crunchyroll-purge-credentials' argument without success and I have also checked my account on the actual website and it works without any issues.

I am on the latest release of streamlink (1.5.0) and Windows 10.

### Reproduction steps / Explicit stream URLs to test

1. Pick a URL from crunchyroll to copy-paste
2. Try to stream it with streamlink and correct arguments

### Log output

```
C:\Users\User>streamlink --crunchyroll-username=<userName> --crunchyroll-password=<password> https://www.crunchyroll.com/media-543723 best --loglevel debug
[cli][debug] OS:         Windows 10
[cli][debug] Python:     3.6.6
[cli][debug] Streamlink: 1.5.0
[cli][debug] Requests(2.24.0), Socks(1.7.1), Websocket(0.57.0)
[cli][info] Found matching plugin crunchyroll for URL https://www.crunchyroll.com/media-543723
[cli][debug] Plugin specific arguments:
[cli][debug]  --crunchyroll-username=<userName> (username)
[cli][debug]  --crunchyroll-password=<password> (password)
[utils.l10n][debug] Language code: en_SE
[plugin.crunchyroll][debug] Creating session with locale: en_SE
[plugin.crunchyroll][debug] Session created with ID: <SID>
[plugin.crunchyroll][debug] Attempting to login using username and password
error: Unable to validate API response: Unable to validate key 'auth': Type of None should be 'str' but is 'NoneType'
```

### Additional comments, screenshots, etc.

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by mkbloke** [comment]
Duplicates #3153.  This is already fixed in master via #3150.

**Comment by EricJSimon** [comment]
Apologies. I forgot to check the closed issues also.
